apps/native-component-list/src/screens/AV/Player.tsx

```diff
@@ -175,6 +176,8 @@ export default function Player(props: Props) {
         {_renderReplayButton()}
       </View>
 
+      <Text>{props.metadata.title || ''}</Text>
```

Comment:
I guess this is more semantically correct 😅 
```suggestion
      <Text>{props.metadata.title ?? ''}</Text>
```

Reply:
Changed!